kalibro_client/processor/metric_result.py

```diff
@@ -0,0 +1,75 @@
+from kalibro_client.base import attributes_class_constructor, \
+    entity_name_decorator
+from kalibro_client.processor.base import Base
+from kalibro_client.processor import Repository
+import kalibro_client.miscellaneous.date_metric_result
+
+
+
+@entity_name_decorator
+class MetricResult(attributes_class_constructor('MetricResultAttrs',
+                                                (('module_result_id', None),)),
+                   Base):
+
+    def __init__(self, value=None, metric_configuration_id=None,
+                 aggregated_value=None, *init_args, **init_kwargs):
+        super(MetricResult, self).__init__(*init_args, **init_kwargs)
+        self.value = float(aggregated_value) if value is None else float(value)
```

Comment:
This validation is not needed. We should use the validation on the value's setter.